### PR TITLE
Prabir/possible models

### PR DESCRIPTION
**Pull request by prabirmsp**
Define course models in Models.md

### CODE REVIEW

File: Models.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Course Models
+
+### Semester
+* `academic_year` : string
+* `semester` : string (ex. fall or spring)
+* `courses_offered`: [Offering]
+
+### Course
+* `number` : integer/string
```
Comment (alexthemitchell) [inline Models.md]: Is this supposed to be like a unique identifier? How is it different from `course_id`?

File: Models.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* `course_id` : string (ex. "TUT 100" / "CSC 151")
+* `title` : string
+* `description` : string
+* `required` : string
```
Comment (alexthemitchell) [inline Models.md]: I'm still in favor of a boolean, but will defer to a second opinion.

File: Models.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* `offerings` : [Offering]
+
+### Offering
+* `course` : Course
+* `professor` : Professor
+* `time` : string
+* `semester` : Semester
+* `students_planned` : [Student]
+* `capacity` : integer
+
```
Comment (alexthemitchell) [inline Models.md]: Nit: Extraneous newline

File: Models.md (hunk 1 above)
Comment (prabirmsp) [inline Models.md]: removed `number`

File: Models.md (hunk 3 above)
Comment (prabirmsp) [inline Models.md]: fixed

**Pull request merged**